test/e2e/framework/framework.go

```diff
@@ -248,6 +250,8 @@ func (f *Framework) BeforeEach() {
 func (f *Framework) AfterEach() {
 	RemoveCleanupAction(f.cleanupHandle)
 
+	printMySQLPodContainerLogs(f)
```

Comment:
Needs a more accurate name. It's not printing, mysql pod container logs, it's retrieving, storing, dumping all container logs really...